### PR TITLE
Elasticsearch update to 1.7.2

### PR DESCRIPTION
**Pull request by gerardorochin**
Pull Request: docker-library/elasticsearch/pull/56

### CODE REVIEW

**Comment by gerardorochin** [comment]
Is pending to accept docker-library/elasticsearch/pull/56

File: library/elasticsearch [hunk 1]
```diff
@@ -12,10 +12,10 @@
...
 1.6: git://github.com/docker-library/elasticsearch@546a3d48ac776256ab4b78a7136cc57ef0e8ddab 1.6
 
-1.7.1: git://github.com/docker-library/elasticsearch@546a3d48ac776256ab4b78a7136cc57ef0e8ddab 1.7
-1.7: git://github.com/docker-library/elasticsearch@546a3d48ac776256ab4b78a7136cc57ef0e8ddab 1.7
-1: git://github.com/docker-library/elasticsearch@546a3d48ac776256ab4b78a7136cc57ef0e8ddab 1.7
-latest: git://github.com/docker-library/elasticsearch@546a3d48ac776256ab4b78a7136cc57ef0e8ddab 1.7
+1.7.2: git://github.com/docker-library/elasticsearch@aadd13a371dc4226eba7409c59b9d6cffb227f78 1.7
+1.7: git://github.com/docker-library/elasticsearch@aadd13a371dc4226eba7409c59b9d6cffb227f78 1.7
+1: git://github.com/docker-library/elasticsearch@aadd13a371dc4226eba7409c59b9d6cffb227f78 1.7
+latest: git://github.com/docker-library/elasticsearch@aadd13a371dc4226eba7409c59b9d6cffb227f78 1.7
```
Comment (tianon) [inline library/elasticsearch]: These need to be updated to `18527ddda24e2f765c18c925f8970b5570882a3c` now. :+1:

(As seen in the output of `./generate-stackbrew-library.sh` when run within the `elasticsearch` repo.)

**Comment by gerardorochin** [comment]
Fixed

**Comment by yosifkit** [comment]
LGTM, Build test of #1047; 7570636143102d96e9b86bb5944f8299de1714d6 (`elasticsearch`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/7570636143102d96e9b86bb5944f8299de1714d6/library/elasticsearch"
$ bashbrew build "$url"
Fetching elasticsearch (git://github.com/docker-library/elasticsearch) ...
Processing elasticsearch:1.3.9 ...
Processing elasticsearch:1.3 ...
Processing elasticsearch:1.4.5 ...
Processing elasticsearch:1.4 ...
Processing elasticsearch:1.5.2 ...
Processing elasticsearch:1.5 ...
Processing elasticsearch:1.6.2 ...
Processing elasticsearch:1.6 ...
Processing elasticsearch:1.7.2 ...
Processing elasticsearch:1.7 ...
Processing elasticsearch:1 ...
Processing elasticsearch:latest ...
Processing elasticsearch:2.0.0-beta1 ...
Processing elasticsearch:2.0.0 ...
Processing elasticsearch:2.0 ...
Processing elasticsearch:2 ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing elasticsearch:1.3.9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing elasticsearch:1.4.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing elasticsearch:1.5.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing elasticsearch:1.6.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing elasticsearch:1.7.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing elasticsearch:2.0.0-beta1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
```

**Comment by tianon** [comment]
LGTM

**Pull request merged**